--- origsrc/clutter-1.0.2/clutter/Makefile.am	2009-08-14 10:11:22.000000000 -0500
+++ src/clutter-1.0.2/clutter/Makefile.am	2009-09-08 00:02:15.066559900 -0500
@@ -284,7 +284,7 @@ Clutter-@CLUTTER_API_VERSION@.typelib: C
 # Clutter.gir to Cogl.gir
 Cogl-@CLUTTER_API_VERSION@.gir: Clutter-@CLUTTER_API_VERSION@.gir cogl/Cogl-@CLUTTER_API_VERSION@.gir
 	shlib=`sed -n 's/.*shared-library="\([^"]*\)".*/\1/p' < Clutter-@CLUTTER_API_VERSION@.gir` ; \
-	sed "s/shared-library=\"clutter-cogl\"/shared-library=\"$$shlib\"/"< cogl/Cogl-@CLUTTER_API_VERSION@.gir > $@
+	sed "s/shared-library=\"\"/shared-library=\"$$shlib\"/"< cogl/Cogl-@CLUTTER_API_VERSION@.gir > $@
 
 BUILT_GIRSOURCES += Cogl-@CLUTTER_API_VERSION@.gir
 
@@ -293,7 +293,7 @@ if LOCAL_JSON_GLIB
 # for Cogl above
 ClutterJson-@CLUTTER_API_VERSION@.gir: Clutter-@CLUTTER_API_VERSION@.gir json/ClutterJson-@CLUTTER_API_VERSION@.gir
 	shlib=`sed -n 's/.*shared-library="\([^"]*\)".*/\1/p' < Clutter-@CLUTTER_API_VERSION@.gir` ; \
-	sed "s/shared-library=\"clutter-json\"/shared-library=\"$$shlib\"/"< json/ClutterJson-@CLUTTER_API_VERSION@.gir > $@
+	sed "s/shared-library=\"\"/shared-library=\"$$shlib\"/"< json/ClutterJson-@CLUTTER_API_VERSION@.gir > $@
 
 BUILT_GIRSOURCES += ClutterJson-@CLUTTER_API_VERSION@.gir
 endif
--- origsrc/clutter-1.0.2/clutter/cogl/Makefile.am	2009-07-29 08:32:09.000000000 -0500
+++ src/clutter-1.0.2/clutter/cogl/Makefile.am	2009-09-07 23:42:54.995084300 -0500
@@ -75,7 +75,7 @@ Cogl-@CLUTTER_API_VERSION@.gir: $(INTROS
 		--c-include='cogl/cogl.h' \
 		--include=GL-1.0 \
 		--include=GObject-2.0 \
-		--library=clutter-cogl \
+		--library=libclutter-cogl.la \
 		--libtool="$(top_builddir)/libtool" \
 		--pkg gobject-2.0 \
 		--output $@ \
--- origsrc/clutter-1.0.2/clutter/json/Makefile.am	2009-08-14 10:11:23.000000000 -0500
+++ src/clutter-1.0.2/clutter/json/Makefile.am	2009-09-07 23:47:20.756645000 -0500
@@ -42,7 +42,7 @@ ClutterJson-@CLUTTER_API_VERSION@.gir: $
 	$(INTROSPECTION_SCANNER) -v --namespace ClutterJson --nsversion=@CLUTTER_API_VERSION@ \
 		--add-include-path=$(srcdir) --add-include-path=. \
 		--include=GObject-2.0 \
-		--library=clutter-json \
+		--library=libclutter-json.la \
 		--libtool="$(top_builddir)/libtool" \
 		--pkg gobject-2.0 \
 		--output $@ \
