--- origsrc/clutter-1.8.2/configure.ac	2011-11-17 03:16:21.823487700 -0600
+++ src/clutter-1.8.2/configure.ac	2011-11-17 03:15:29.832514000 -0600
@@ -83,7 +83,12 @@ CLUTTER_LT_CURRENT=lt_current
 
 AC_MSG_CHECKING([if building for some Win32 platform])
 AS_CASE([$host],
-        [*-*-mingw*|*-*-cygwin*],
+        [*-*-cygwin*],
+        [
+          CLUTTER_LT_LDFLAGS="$CLUTTER_LT_LDFLAGS -no-undefined"
+        ],
+
+        [*-*-mingw*],
         [
           CLUTTER_LT_LDFLAGS="$CLUTTER_LT_LDFLAGS -no-undefined"
           platform_win32=yes
